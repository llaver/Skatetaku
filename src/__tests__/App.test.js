import React from 'react'
import {shallow} from 'enzyme'
import Component from '../App'

describe('App', () => {
  it('should match its empty snapshot', () => {
    const shallowed = shallow(<Component foo="baz" />)
    expect(shallowed).toMatchSnapshot()
  })
})