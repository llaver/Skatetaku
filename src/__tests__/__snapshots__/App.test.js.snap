// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match its empty snapshot 1`] = `
<styled.div>
  <h1>
    Hello World ðŸ“¦ ðŸš€
  </h1>
</styled.div>
`;